applicationUi/src/main/java/edu/wpi/grip/ui/pipeline/AddSourceView.java

```diff
@@ -3,8 +3,8 @@
 import com.google.common.eventbus.EventBus;
 import edu.wpi.grip.core.events.FatalErrorEvent;
 import edu.wpi.grip.core.events.SourceAddedEvent;
-import edu.wpi.grip.core.sources.ImageFileSource;
 import edu.wpi.grip.core.sources.CameraSource;
+import edu.wpi.grip.core.sources.ImageFileSource;
```

Comment:
Please get rid of this commit noise.  This will just make merging stuff more frustrating.